.github/workflows/build.yml

```diff
@@ -17,7 +17,7 @@ env:
   CARGO_TERM_COLOR: always
   CARGO_INCREMENTAL: '0'
   CARGO_PROFILE_DEV_DEBUG: '0'
-  RUST_TOOLCHAIN_VERSION: "1.79.0"
```

Comment:
We should maybe bump to `1.80.1` here and below as well.

Reply:
Good point, 7fe48c9